### ISSUE TITLE
Per-boss loadouts don't work

### ISSUE THREAD

**Issue by filliph** [derfloh205/LoadoutReminder#8]
Targetting a boss with a per-boss loadout set does not prompt you to switch it. Only the "All Raids" settings apply.

**Comment by filliph** [comment]
@derfloh205 any word on when this will be fixed? The addon hasn't functioned since 4.2.0.

**Comment by filliph** [comment]
Debug info from v6 (this still doesn't work):

```
[22:19:25] /lor instance 
[22:19:25] InstanceType: RAID 
[22:19:25] IsInInstance: true 
[22:19:25] - id: 2549 
[22:19:25] - Raid: AMIRDRASSIL 
[22:19:25] Difficulty: HEROIC 
[22:19:25] -
```

```
[22:20:10] /lor npc 
[22:20:10] NPC: Smolderon 
[22:20:10] - id: 200927
```

![image](https://github.com/derfloh205/LoadoutReminder/assets/9424286/cc0b700c-2c20-4c27-83ad-ce669066d753)

![image](https://github.com/derfloh205/LoadoutReminder/assets/9424286/8624d0a3-fd88-46f7-9468-b4f22d7a8a97)

Nothing is shown on screen to suggest I should switch builds.
